### ISSUE TITLE
Kudos for this package, and a quick heads-up

### ISSUE THREAD

**Issue by mceachen** [raineorshine/npm-check-updates#770]
First off: thanks so much, @raineorshine , for developing and maintaining `npm-check-updates`: :tada: 

The quick heads-up: it seems that version 10+ of `npm-check-updates` dropped support for Node v10. Is that by design?

- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 10.17`

---------------------------

## Steps to Reproduce

1. Use node v10.23.0
2. Try to install `npm-check-updates`.

## Current Behavior

```sh
 npm i -g npm-check-updates
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
/home/mrm/.nvm/versions/node/v10.23.0/bin/npm-check-updates -> /home/mrm/.nvm/versions/node/v10.23.0/lib/node_modules/npm-check-updates/bin/cli.js
/home/mrm/.nvm/versions/node/v10.23.0/bin/ncu -> /home/mrm/.nvm/versions/node/v10.23.0/lib/node_modules/npm-check-updates/bin/cli.js
npm WARN notsup Unsupported engine for fp-and-or@0.1.2: wanted: {"node":">=12"} (current: {"node":"10.23.0","npm":"6.14.8"})
npm WARN notsup Not compatible with your version of node/npm: fp-and-or@0.1.2
```

## Expected Behavior

I didn't expect Node v10 support to be dropped in npm-check-updates v10+. FWIW, version 9 installs fine on Node 10:

```sh
npm i -g npm-check-updates@9
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
/home/mrm/.nvm/versions/node/v10.23.0/bin/npm-check-updates -> /home/mrm/.nvm/versions/node/v10.23.0/lib/node_modules/npm-check-updates/bin/cli.js
/home/mrm/.nvm/versions/node/v10.23.0/bin/ncu -> /home/mrm/.nvm/versions/node/v10.23.0/lib/node_modules/npm-check-updates/bin/cli.js
+ npm-check-updates@9.2.4
removed 1 package and updated 2 packages in 5.351s
```

Node v10 EOLs 2021-04-30, according to https://nodejs.org/en/about/releases/

Again: thanks for your work on this package!

Also: I'm providing free "plus" licenses of [PhotoStructure](https://PhotoStructure.com) to all devs and maintainers of all direct and critical tooling that PhotoStructure uses (there are a [few](https://photostructure.com/legal/3rd-party-licenses.txt)): I'd be delighted if you could glean value from that: just send me an email (address is on the website).

**Comment by raineorshine** [comment]
Thanks for bringing this to my attention! This was a mistake I made in the [fp-and-or](https://github.com/raineorshine/fp-and-or) dependency. 

I fixed the engine and released `v10.2.2`.

**Comment by XhmikosR** [comment]
Closing since this seems to be fixed in >= v10.2.2
